### PR TITLE
Remove offset on GOES obs converter

### PR DESCRIPTION
**Pull request by braczka**
## Description:
I removed the satellite channel offset within GOES ABI observation converter such that the wavelength in the obs_seq.out match that
within the RTTOV coefficient file.

I also improved the documentation both within the GOES obs converter and the RTTOV module to alert the user to potential
mismatch issues.

### Fixes issue
#731 

### Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Documentation update

### Documentation changes needed?

- [ ] My change requires a change to the documentation.
  - [X] I have updated the documentation accordingly.

### Tests
I have performed offline checks with the GOES obs converter, then used those obs_seq.out files to run filter
to confirm functionality.

## Checklist for merging

- [ ] Updated changelog entry
- [X] Documentation updated
- [ ] Update conf.py

## Checklist for release
- [ ] Merge into main
- [ ] Create release from the main branch with appropriate tag
- [ ] Delete feature-branch

## Testing Datasets

- [ ] Dataset needed for testing available upon request
- [ ] Dataset download instructions included
- [X] No dataset needed

### CODE REVIEW

**Comment by hkershaw-brown** [comment]
Hi Brett, can you provide 2 GOES ABI observation files for testing since we also have this issue outstanding: https://github.com/NCAR/DART/issues/579
Thanks!

**Comment by hkershaw-brown** [comment]
two would be great since the other bug is "bug: GOES converter does not handle multiple input files"
Cheers,
Helen

**Comment by braczka** [comment]
> two would be great since the other bug is "bug: GOES converter does not handle multiple input files" Cheers, Helen

Done

**Comment by hkershaw-brown** [comment]
If you build the converter with rttov13, then run, it errors out on the namelist.
I think this all the rrtov12 options in the input.nml &obs_def_rttov_nm

```
hkershaw@dec2158:/glade/derecho/scratch/hkershaw/DART/pull_reqeuests/pull_732/observations/obs_converters/GOES/work(GOES_obs_converter)$ ./convert_goes_ABI_L1b 
 
 --------------------------------------
 Starting ... at YYYY MM DD HH MM SS = 
                 2024  9 11  9 57 16
 Program convert_goes_ABI_L1b
 --------------------------------------
 
  set_nml_output Echo NML values to log file only
  goes_load_abi_map  Now loading L1C data from file OR_ABI-L1b-RadF-M6C08_G16_s2
 0211841720210_e20211841729518_c20211841729575.nc
 ERROR FROM:
  source : utilities_mod.f90
  routine: check_namelist_read
  message:  INVALID NAMELIST ENTRY:    cfrac_data             = .true. in nameli
 st obs_def_rttov_nml
```

**Comment by braczka** [comment]
Sorry -- the input.nml file got reverted to the default.  The input.nml I used for the GOES conversion is here:
/glade/work/bmraczka/DART/observations/obs_converters/GOES/work/input.nml.GOES_converter

**Review by hkershaw-brown — CHANGES_REQUESTED** [review]
see default input.nml comment

**Comment by braczka** [comment]
> see default input.nml comment

Changes made to make input.nml compatible to run the converter.  Should be ready for approval.

**Comment by hkershaw-brown** [comment]
> > see default input.nml comment
> 
> Changes made to make input.nml compatible to run the converter. Should be ready for approval.

ok I haven't actually looked at this pull request since trying to run it. I'll update to main and take a look.

**Review by hkershaw-brown — CHANGES_REQUESTED** [review]
Hi Brett,

A few things
There's a broken link in to the GEOS converter, typo $ vs ~

convert_goes_ABI_L1b is missing from DART/.gitignore

Some of the documentation on rttov database vs. rttov coeffiencts is not that clear (e.g. the database is a DART file)
The vertical location for radiance obs I think would benefit from mentioning the namelist option to set a vertical location.

ps. I'm waiting for a Derecho job to actually run the code. 

Cheers,
Helen

File: observations/forward_operators/obs_def_rttov_mod.rst [hunk 1]
```diff
@@ -46,19 +46,114 @@ Although a model may not have the necessary inputs by itself,
...
+
+  **It is imperative that the user confirms the satellite integer metadata matches the
+  appropriate RTTOV coefficient (parameter) file.  See next section for more information.**
+
+RTTOV coefficient files
+-----------------------
+
+The RTTOV coefficent file contains the appropriate parameter values for a specific satellite
+radiance observation. The RTTOV database file (``rttov_sensor_db.csv``) refers to the coefficent
+file.  For the ``HIMAWARI_9_AHI_RADIANCE`` observation type, for example, the following information
```
[the remark this answers is not in the document]
Comment (braczka) [reply observations/forward_operators/obs_def_rttov_mod.rst]: > The rttov_sensor_db.csv a DART file, correct? I would be good to make that clear. At the moment it reads like the RTTOV database file is something you should be getting from RTTOV.
> 
> The namelist option:
> 
> rttov_sensor_db_file	character(len=512)	The location of the RTTOV sensor database. The format for the database is a comma-separated file. The columns of the database are the DART observation-kind, the platform/satellite/sensor ID, the observation type, the coefficient file, and a comma-separated list of RTTOV channels to use for this observation type.
> looking at the code the channel list is optional:
> 
> https://github.com/NCAR/DART/blob/464aa57a1261fa098e5f0f999bad41a464f8d7db/observations/forward_operators/obs_def_rttov13_mod.f90#L1059
> 
> there are no channels in the rttov_sensor_db.csv sensor file in the repo:
> 
> ```
> NOAA_1_VTPR1,1,1,7,ir,rtcoef_noaa_1_vtpr1.dat
> NOAA_2_VTPR1,1,2,7,ir,rtcoef_noaa_2_vtpr1.dat
> NOAA_3_VTPR1,1,3,7,ir,rtcoef_noaa_3_vtpr1.dat
> ...
> ```
> 
> are users supposed to edit rttov_sensor_db.csv or leave it as is?

The recommendation is to leave as is, and allow for all channels to be available, and the correct channel parameters are specified through the obs_seq.out channel metadata.  Not clear to me if there would be any application where excluding certain channels through the DART database file would be appropriate.

[the remark this answers is not in the document]
Comment (braczka) [reply observations/forward_operators/obs_def_rttov_mod.rst]: @hkershaw-brown,   I think I addressed all your review suggestions.  Check through these docs changes, and make sure the converter code doesn't give you any issues.

**Review by hkershaw-brown — APPROVED** [review]
looks great, approved. 

todo: note in release notes, this will affect people using subsetted rttov files.

**Pull request merged**